components/fhir-core/src/main/java/org/wso2/healthcare/codegen/tool/framework/fhir/core/FHIRTool.java

```diff
@@ -18,51 +18,45 @@
 
 package org.wso2.healthcare.codegen.tool.framework.fhir.core;
 
-import io.swagger.v3.oas.models.OpenAPI;
-import io.swagger.v3.parser.OpenAPIV3Parser;
-import org.apache.commons.io.IOUtils;
 import org.wso2.healthcare.codegen.tool.framework.commons.config.ToolConfig;
 import org.wso2.healthcare.codegen.tool.framework.commons.core.AbstractTool;
 import org.wso2.healthcare.codegen.tool.framework.commons.core.Tool;
 import org.wso2.healthcare.codegen.tool.framework.commons.core.ToolContext;
 import org.wso2.healthcare.codegen.tool.framework.commons.exception.CodeGenException;
-import org.wso2.healthcare.codegen.tool.framework.fhir.core.common.FHIRSpecificationData;
 
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.nio.charset.StandardCharsets;
 import java.util.HashMap;
 import java.util.Map;
 
 /**
- * This class represents the FHIR tool library which will hold the contexual data and utilities for the
+ * This class represents the FHIR tool library which will hold the contextual data and utilities for the
  * FHIR tool implementations to generate artifacts.
  */
 public class FHIRTool extends AbstractTool {
 
+    private static String fhirVersion = "r4";
```

Comment:
Shall we make this an instance variable since we're setting the value in the constructor?